xrootd/cmd/xrd-srv/handler.go

```diff
@@ -127,8 +127,8 @@ func (h *handler) Open(sessionID [16]byte, request *open.Request) (xrdproto.Mars
 
 // Close implements server.Handler.Close.
 func (h *handler) Close(sessionID [16]byte, request *xrdclose.Request) (xrdproto.Marshaler, xrdproto.ResponseStatus) {
-	sess, ok := h.sessions.Load(sessionID)
-	if !ok {
+	sess, dup := h.sessions.Load(sessionID)
+	if !dup {
```

Comment:
actually, I was suggesting `s/ok/dup/` only for the one at line 114.
the others were fine with `ok` :)
(sorry.)